packages/editor-ui/src/views/SettingsLdapView.vue

```diff
@@ -294,7 +294,7 @@ export default mixins(showMessage).extend({
 				loginLabel: formInputs.values.loginLabel,
 				connectionUrl: formInputs.values.serverAddress,
 				allowUnauthorizedCerts: formInputs.values.allowUnauthorizedCerts,
-				connectionPort: formInputs.values.port,
+				connectionPort: +formInputs.values.port,
```

Comment:
I'm not sure why this is needed. The value is already a number because it's already defined as `type: 'number'`. Did you see this being a string while debugging?

Reply:
When I tested it, it was returning a string, and I thought it was a bug with the form component. But I just tested it again, and it's returning a number. I wonder if we fixed that after I tested it.